unit_tests/UnitTestFieldRegistry.C

```diff
@@ -0,0 +1,78 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+#include "gtest/gtest.h"
+#include "stk_mesh/base/MeshBuilder.hpp"
+#include "FieldRegistry.h"
+#include <memory>
+#include <stdexcept>
+
+namespace sierra {
+namespace nalu {
+namespace {
+
+class FieldRegistryTest : public testing::Test
+{
+protected:
+  void SetUp()
+  {
+    stk::mesh::MeshBuilder builder(MPI_COMM_WORLD);
+    bulk_ = builder.create();
+    key_ = "velocity";
+  }
+  stk::mesh::MetaData& meta() { return bulk_->mesh_meta_data(); }
+  std::shared_ptr<stk::mesh::BulkData> bulk_;
```

Comment:
still don't think we need `bulk_`

Reply:
@psakievich What Timothy is saying is that you could use MeshBuilder::create_meta_data() to create a MetaData directly, without needing to create or hold a BulkData.

Apologies if that was already explicitly stated earlier in these reviews/comments.

Reply:
Ah so use that instead of the `create()` method?